packages/dashboards/tests/dummy/config/environment.js

```diff
@@ -25,7 +25,7 @@ module.exports = function(environment) {
 
     navi: {
       FEATURES: {
-        enableDirectory: true,
+        enableDirectory: false,
```

Comment:
Is there an app that doesn't use `directory`?

Reply:
Zig doesn't